ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx

```diff
@@ -1,18 +1,18 @@
-import { useMemo } from 'react';
 import { TextInput } from '@percona/ui-lib';
 import { FormDialog } from 'components/form-dialog';
-import TlsCheckbox from 'components/tls-checkbox';
 import TlsAlert from 'components/tls-alert';
+import TlsCheckbox from 'components/tls-checkbox';
+import { useMemo } from 'react';
+import { AutoCompleteSelectAll } from '../../../../components/auto-complete-select-all/auto-complete-select-all';
+import { useNamespaces } from '../../../../hooks/api/namespaces/useNamespaces';
+import { Messages } from '../monitoring-endpoints.messages';
 import {
   CreateEditEndpointModalProps,
   EndpointFormFields,
   EndpointFormType,
   endpointDefaultValues,
   getEndpointSchema,
 } from './create-edit-modal.types';
-import { Messages } from '../monitoring-endpoints.messages';
```

Comment:
the same question about messages, why we change the position of import? is this result of some command?

Reply:
Its just the Prettier extension and formatting on save